### PR TITLE
feat(random): add random endpoint

### LINKED ISSUE #19: Need random endpoint
Return random info

### PR DESCRIPTION
**Pull request by dp0973**
close #19

### CODE REVIEW

**Review by SaidBySolo — CHANGES_REQUESTED** [review]
Need test code

File: tests/test_random.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from sanic_testing.manager import TestManager  # type:ignore
+
+url = "/v5/api/hitomi/random/"
```
Comment (SaidBySolo) [inline tests/test_random.py:3]:
```suggestion
url = "/v5/api/hitomi/random"
```

**Review by SaidBySolo — CHANGES_REQUESTED** [review]

**Comment by SaidBySolo** [comment]
@dp0973 Check test

File: heliotrope/database/mongo.py [hunk 2]
```diff
@@ -54,5 +54,11 @@ async def find_info(self, index_id: int) -> dict[str, Any]:
             await self.__collection.find_one({"index": index_id}, {"_id": 0}),
         )
 
+    async def find_random_info(self) -> dict[str, Any]:
+        return cast(
+            dict[str, Any],
+            await self.__collection.aggregate({"$sample": {"size": 1}}),
```
Comment (SaidBySolo) [inline heliotrope/database/mongo.py:60]: Exclude the _id key

File: tests/test_random.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+from sanic_testing.manager import TestManager  # type:ignore
+
+url = "/v5/api/hitomi/random"
+
+
+def test_random(app: TestManager):
+    _, response = app.test_client.get()
```
Comment (SaidBySolo) [inline tests/test_random.py:7]: Need url

**Review by SaidBySolo — CHANGES_REQUESTED** [review]
We need two test codes. One for normal requests and one for requests not found.

File: tests/test_random.py (hunk 3 above)
Comment (ombe1229) [inline tests/test_random.py:7]:
```suggestion
    _, response = app.test_client.get(url)
```

File: heliotrope/database/mongo.py [hunk 4]
```diff
@@ -54,5 +54,11 @@ async def find_info(self, index_id: int) -> dict[str, Any]:
             await self.__collection.find_one({"index": index_id}, {"_id": 0}),
         )
 
+    async def find_random_info(self) -> dict[str, Any]:
+        return cast(
+            dict[str, Any],
+            await self.__collection.aggregate({"$sample": {"size": 1}}, {"_id": False}),
```
Comment (SaidBySolo) [inline heliotrope/database/mongo.py:60]:
```suggestion
            await self.__collection.aggregate({"$sample": {"size": 1}}, {"_id": 0}),
```

**Review by SaidBySolo — CHANGES_REQUESTED** [review]

**Review by SaidBySolo — APPROVED** [review]
LGTM

Thx for work :)

**Pull request merged**